tests/pageObjects/AddPromotion.js

```diff
@@ -0,0 +1,58 @@
+class AddPromotion{
+    constructor(){
+        this.adminIconSelector = "#account-button";
+        this.adminPanelSelector = "//div[@id='link-to-account']/a[2]";
+        this.promotionTabSelector = "//a[@href='#sidebar-promotions']";
+        this.promotionSubTabSelector = "//a[@href='/admin/promotions']";
+
+        this.nameFieldSelector = "#promotion_name";
+        this.codeFieldSelector = "#promotion_code";
+        this.couponCodeCheckboxSelector = "#promotion_generate_code";
+        this.pathFieldSelector = "#promotion_path";
+        this.advertiseCheckboxSelector = "#promotion_advertise";
+        this.descriptionFieldSelector = "#promotion_description";
+        this.limitFieldSelector = "#promotion_usage_limit";
+        this.startDateIconSelector = "//div[@id='starts_at_field']/div/div/span[@class='input-group-text']";
+        this.startDateSelector = "(//div[@class='dayContainer'])[1]/span[1]";
+        this.headerSelector = "//div[@class='col text-right p-0']"
+        this.endDateIconSelector = "//div[@id='expires_at_field']/div/div/span[@class='input-group-text']";
```

Comment:
Try to make the selector shorter.
```suggestion
        this.endDateIconSelector = "//div[@id='expires_at_field']/div/div/span[@class='input-group-text']";
```